.github/workflows/deno.yml

```diff
@@ -56,10 +56,30 @@ jobs:
 
             - uses: denoland/setup-deno@main
               with:
-                  deno-version: v1.21
+                  deno-version: v1.x
 
             - name: Cache Dependencies
-              run: npm run cache
+              run: deno cache src/mod.ts
 
             - name: Run Tests
-              run: npm test
+              run: deno task test
+
+    coverage:
+        runs-on: ubuntu-latest
+        steps:
+            - name: Setup repo
+              uses: actions/checkout@v3
+              with:
+                  fetch-depth: 0
+
+            - uses: denoland/setup-deno@main
+              with:
+                  deno-version: v1.x
+
+            - name: Create coverage files
+              run: deno task coverage
+
+            - name: Collect coverage
+              uses: codecov/codecov-action@v1.0.10 # upload the report on Codecov
+              with:
+                  file: ./coverage.lcov
```

Comment:
I don't know if it is better or affects the speed, but you can collect the coverage, too, with one `checkout` and `setup-deno`, both in a single job.

Reply:
I'm not sure what you mean, how can a single job perform both the checkout and the Deno installation?

Reply:
I mean doing the coverage collection in the `test` job, but that seems it is going to be not helpful as it is run 3 times and producing the same result.